pkg/utils/utils.go

```diff
@@ -656,3 +656,17 @@ func GetPhysPortName(name string) (string, error) {
 	}
 	return "", nil
 }
+
+func GetVFNameFromIndex(pfPciAddress string, vfID int) (string, error) {
+	glog.V(2).Infof("GetVFNameFromIndex(): PF %s, vfID %d", pfPciAddress, vfID)
```

Comment:
nit: dont think we need logging in this util method

Reply:
right, this one is going to be removed when sriovnet PR merged

Reply:
which PR is that ? can you paste a link ?

Reply:
https://github.com/Mellanox/sriovnet/pull/36